pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1373,3 +1373,267 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func TestSlowStartBatch(t *testing.T) {
+	fn := func() error {
+		return nil
+	}
+	errFn := func() error {
+		return fmt.Errorf("fake error")
```

Comment:
Declare this error as a variable outside the function, and refer to the same variable both here and when setting `expectedErr`. It may not always be the case in the future that two values returned from `fmt.Errorf()` with the same string will compare equal.